### DISCUSSION TITLE
Update qdomyos on rasberry pi 0

### DISCUSSION THREAD

**Discussion by urbanspike** [cagnulein/qdomyos-zwift#1012] [General]
Hello,

How are you ?
I have an old version of the qdomyos application on my pi zero (2 years old?), how can I update it to the latest version? What command to pass on the pi to get the latest version of qdomyos?

Thank you ++

**Comment by cagnulein** [comment]
hi @urbanspike do a 

git pull
make -j1
and you are ready to go

let me know

**Reply by urbanspike** [reply]
i have 
Cd home/pi/qdomyos-zwift/src
Git pull
Make -j1
Error on usr/lib/qt5/quake -o maléfice qdomyos-zwift.pro
Project error : unknown module in qt....
Any idea ?

**Reply by cagnulein** [reply]
Please post the full log

**Reply by urbanspike** [reply]
Error on usr/lib/qt5/quake -o maléfice qdomyos-zwift.pro
Project error : unknown module in qt : networkauth websockets texttospeech location multimedia quickcontrols2

**Reply by cagnulein** [reply]
Run this

sudo apt install git qtquickcontrols2-5-dev libqt5bluetooth5 libqt5widgets5 libqt5positioning5 libqt5xml5 qtconnectivity5-dev qtpositioning5-dev libqt5charts5-dev libqt5charts5 qt5-assistant libqt5networkauth5-dev libqt5websockets5-dev qml-module* libqt5texttospeech5-dev libqt5texttospeech5 libqt5location5-plugins qtlocation5-dev qtmultimedia5-dev libqt5multimediawidgets5 libqt5multimedia5-plugins libqt5multimedia5

**Reply by urbanspike** [reply]
Not good
I have delete qdomyos-zwift folder
Sudo apt update : no error
Sudo apt upgrade : no error
Sudo apt install git qtquick...... no error
Git clone : no error
Cd qdomyos-zwift
Git submodule update for smtp and qmdnsengine : no error
Qmake : empty filename passes to fonction on several line ??
Make error i.post a photo

**Comment by urbanspike** [comment]
![20221030_215208](https://user-images.githubusercontent.com/20811473/198901188-8c67483e-f37f-450e-97b2-58c1d0568127.jpg)

**Comment by urbanspike** [comment]
![20221030_220459](https://user-images.githubusercontent.com/20811473/198901481-219782ec-e3f6-4d8a-aa2b-1471617898c7.jpg)

**Reply by cagnulein** [reply]
do a
make clean
